packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx

```diff
@@ -0,0 +1,56 @@
+import { Component, Host, h, Prop, Event, EventEmitter, Method } from '@stencil/core';
+import { getSvg } from '../../common/images/icon-provider';
+/**
+ * Breadcrumb element represents  navigation
+ */
+
+@Component({
+  tag: 'datacom-breadcrumb',
+  styleUrl: 'datacom-breadcrumb.css',
+  shadow: true,
+})
+export class DatacomBreadcrumb {
+  @Prop() link?: string;
+  @Prop() text: string;
+  @Prop() index: number;
+  @Prop({ mutable: true }) selected: boolean;
+  @Event() itemClicked: EventEmitter<number>;
+
+  onClickHandler = () => {
+    this.itemClicked.emit(this.index);
+    this.selected = true;
+  };
+
+  /**
+   *  Function sets the select state of this breadcrumb item.
+   */
+  @Method()
+  async setSelected(selected: boolean): Promise<void> {
```

Comment:
I wouldn't say that a breadcrumb is selected. I'm assuming you mean this is meant to represent that this breadcrumb link is the "current page"?